ADDING_THEMES.md

```diff
@@ -1,5 +1,6 @@
 How to add a new theme to the theme gallery:
 
 1. Take a screenshot of the theme and place it in the /images/gallery/themes folder. Make sure the name of the screenshot is like this: (theme name).png
+NOTE: when loading a theme make sure to use the command with the `full-load` argument like this: `/theme full-load (theme name)`
```

Comment:
The thing is that full-load is overwriting some settings. Which is why we split it into two commands. load would just load the colors of a theme. full-load would load the full theme including things like how to group the roster, to have prefix chars infront of the nicknames, display jid or nickname etc etc.

Maybe adapt the text to include this information so the user understands what effect it will have. You can also see the issue I mentioned earlier where the splitting of load and full-load was discussed.